metadata.json

```diff
@@ -43,8 +43,8 @@
     {
       "operatingsystem": "Fedora",
       "operatingsystemrelease": [
-        "26",
-        "27"
+        "32",
+        "33"
```

Comment:
33 will break test runs because `beaker-hostgenerator` doesn't support it yet. Perhaps leave it out until we can actually test it in CI?